ui/app/src/Views/FilterBar/Views.elm

```diff
@@ -142,7 +141,7 @@ view matchers matcherText backspacePressed =
                             [ text "Custom matcher, e.g."
                             , button
                                 [ class "btn btn-link btn-sm align-baseline"
-                                , onClick (UpdateMatcherText "env=\"production\"" |> MsgForAlertList)
+                                , onClick (UpdateMatcherText "env=\"production\"")
```

Comment:
Small thing: Can we move `"env=\"production\"` to a constant so it stays in sync with line 146?